app/src/main/java/com/gathering/android/common/Extentions.kt

```diff
@@ -0,0 +1,9 @@
+package com.gathering.android.common
+
+const val LOCAL_CONTENT_URL_PREFIX = "content://"
+private const val PHOTO = "photo"
+const val BASE_URL = "https://moyar.dev:8080"
```

Comment:
Move all these constants to the constant file, where all other counts are located.